_sass/components/_details.scss

```diff
@@ -0,0 +1,91 @@
+details.isomer-details summary {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  &:empty {
+    justify-content: flex-end;
+  }
+}
+
+details.isomer-details summary::marker {
+  display: none;
+}
+
+details.isomer-details summary::after {
+  // TODO: Think of a better way to do this
+  content: url("https://raw.githubusercontent.com/atisawd/boxicons/master/svg/solid/bxs-chevron-down.svg");
```

Comment:
if we are set on using boxicons (think we are), we can just link the script and use the web component `box-icon`

Reply:
cc @sehyunidaaa

Reply:
could you explain further ah @seaerchin? exactly what source link are you referring to ah

@sehyunidaaa can confirm using [bi-chevron-down](https://react-icons.github.io/react-icons/search/#q=bichevrondown)

Reply:
ooh my this is a long time ago. that's fine, but why not use the `sgds-icon sgds-icon-chevron-down` that we already have in the template? so we can keep it to one icon style